octodns_edgedns/__init__.py

```diff
@@ -14,7 +14,7 @@
 from octodns.record import Record
 
 # TODO: remove __VERSION__ with the next major version release
-__version__ = __VERSION__ = '1.0.0'
+__version__ = __VERSION__ = '1.1.0'
```

Comment:
Release numbers are bumped as part of dedicated PRs during the release process.
```suggestion
__version__ = __VERSION__ = '1.0.0'
```